rerun_py/rerun_sdk/rerun/archetypes/pinhole_ext.py

```diff
@@ -78,6 +22,62 @@ def __init__(
         focal_length: float | npt.ArrayLike | None = None,
         principal_point: npt.ArrayLike | None = None,
     ) -> None:
+        """
+        Create a new instance of the Pinhole archetype.
+
+        Parameters
+        ----------
+        image_from_camera:
+            Row-major intrinsics matrix for projecting from camera space to image space.
+            The first two axes are X=Right and Y=Down, respectively.
+            Projection is done along the positive third (Z=Forward) axis.
+            This can be specified _instead_ of `focal_length` and `principal_point`.
+        resolution:
+            Pixel resolution (usually integers) of child image space. Width and height.
+            `image_from_camera` projects onto the space spanned by `(0,0)` and `resolution - 1`.
+        camera_xyz:
```

Comment:
TODO: Would be nice to patch these during code-gen if the parameter name matches.